src/server/router-development.ts

```diff
@@ -121,36 +119,11 @@ authRouterDevelopment.get(
             authMethod: auth.profile.authMethod,
           })
         );
-      } catch (error) {}
+      } catch (error) { }
     }
 
     res.clearCookie(OIDC_SESSION_COOKIE_NAME);
     return sendUnauthorized(res);
   }
 );
 
-export const relayDevRouter = express.Router();
-
-relayDevRouter.get(
-  [
-    RelayPathsAllowed.WPI_DOCUMENT_DOWNLOAD,
-    RelayPathsAllowed.VERGUNNINGEN_DOCUMENT_DOWNLOAD,
-    RelayPathsAllowed.LOOD_DOCUMENT_DOWNLOAD,
-    RelayPathsAllowed.BEZWAREN_DOCUMENT,
-    RelayPathsAllowed.WMO_DOCUMENT_DOWNLOAD,
```

Comment:
Heb je deze urls allemaal geverifieerd in de front-end?

Reply:
Allemaal, alleen WPI wijst nog naar 3000 want er staat geen origin in de hrefs. Wat kan ik daarmee doen? Ik kan ze denk ik niet gaan aanpassen toch?